### PR TITLE
Update all alerts to include new alert style

### LINKED ISSUE #432: Clear Connection Between Notification and Error States
As a user, I want consistent and clear notification on areas I need to fix so that I know exactly where to correct my errors.

# Description
Connect top notification bars with appropriate error handling on forms across the site.

# Assumptions
Apply to all forms on the Gitcoin site

# Definition of Done
1. Update notification styles
2. Update notification bars to include icons.
3. Show error/fix states under form fields that need attention. 
4. All forms are using the same error design pattern

# Reviewers
@PixelantDesign

screen shot 2018-02-27 at 12 23 34 am

### PR DESCRIPTION
**Pull request by KennethAshley**
##### Description
Update Alerts

##### Checklist
- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
UI

##### Testing
Pull down and test

##### Refers/Fixes
#432 

#### Examples
Normal
image

Error
image

Info
image

Warning
image

Success
image

### CODE REVIEW

**Comment by KennethAshley** [comment]
@PixelantDesign I talked to @owocki. Handling error states on elements will be a pretty big feature itself so we moved that to another issue (#554)

**Comment by mbeacom** [comment]
@KennethAshley Would you mind resolving the conflicts on this PR?  Also, is this where it needs to be @PixelantDesign ?

**Comment by KennethAshley** [comment]
@mbeacom Done. I think all is fixed. just waiting for approval.

File: app/app/local_settings_old.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+INFURA_KEY = 'TODO' # requied only needed if you use infura
+
+CSRF_TRUSTED_ORIGINS = [
+    'localhost'
+]
+
+# optional: only needed if you run the github management commands
+# Login/logout functionality will not work without defining these fields.
+# Setup instructions: https://github.com/gitcoinco/web#setup-github-oauth2-app-integration
+GITHUB_CLIENT_ID = 'b8b56ad694d752c49c1d'
```
Comment (mbeacom) [inline app/app/local_settings_old.py]: I'm assuming you didn't mean to commit this.  I'd rekey all of your secrets here. @KennethAshley

**Pull request merged**